aries_cloudagent/resolver/default/indy.py

```diff
@@ -35,9 +39,18 @@ async def setup(self, context: InjectionContext):
 
     @property
     def supported_methods(self) -> Sequence[str]:
-        """Return supported methods of Indy DID Resolver."""
+        """
+        Return supported methods of Indy DID Resolver.
+
+        DEPRECATED: Use supported_did_regex instead.
+        """
         return ["sov"]
 
+    @property
+    def supported_did_regex(self) -> Pattern:
+        """Return supported_did_regex of Indy DID Resolver."""
+        return re.compile(IndyDID.PATTERN)
```

Comment:
I think it's worth creating this globally and returning a pre-compiled `Pattern` instance.